src/components/calcite-list-item/readme.md

```diff
@@ -0,0 +1,3 @@
+# calcite-list-item
+
```

Comment:
This will need some base info on the component too. 

I think we can drop the item's usage file since the list one would cover it.